crates/neon-build/src/lib.rs

```diff
@@ -31,6 +31,43 @@ cfg_if! {
                 println!("cargo:rustc-cdylib-link-arg=/DELAYLOAD:node.exe");
             }
         }
+    } else if #[cfg(windows)] {
+        // ^ automatically not neon-sys
+        use std::io::{Error, ErrorKind, Result};
+        use std::process::Command;
+
+        fn node_version() -> Result<String> {
+            let output = Command::new("node").arg("-v").output()?;
+            let stdout = String::from_utf8(output.stdout).map_err(|error| {
+                Error::new(ErrorKind::InvalidData, error)
+            })?;
+            Ok(stdout.trim().to_string())
+        }
+
+        fn download_node_lib(version: &str) -> Result<Vec<u8>> {
+            let script = format!(r#"
+                var url = "https://nodejs.org/dist/{version}/win-{arch}/node.lib";
+                // double braces because we're in a format string
```

Comment:
If you wanted to do a non-formatted version, the URL could be sent in as an argument. E.g.,

```sh
$ node -e 'console.log(process.argv)' test
[ '/usr/bin/node', 'test' ]
```

```js
require("https").get(process.argv[1], function (res) { /* ... */ });
```

Reply:
I didn't mind the `{{` myself but I like the idea of using an extra argument :)